Added new defintions (#53)
lhcb/glossary: glossary/M.md

# M

## MaPMT: Multianode-Photomultiplier Tubes {#MaPMT}

A vacuum photodetector, that consists of a set of dynodes under graded potential.

> MaPMTs were installed with Upgrade-1 in RICH 1 and RICH 2.

Reviewer: ```suggestion
MaPMTs were installed in RICH 1 and RICH 2 for Upgrade I.
```

> They replace the previous photodetectors, called [HPD](#HPD).

Reviewer: ```suggestion
They replace the previous photodetectors, called «HPD».
```

> RICH 1 and inner parts of RICH 2 are covered by the 25.4 x 25.4 mm MaPMTs with 64 channels.

Reviewer: ```suggestion
RICH 1 and the inner parts of RICH 2 are covered by the 25.4 x 25.4 mm MaPMTs with 64 channels.
```

> Outer area of RICH 2 is covered with 50.8 x 50.8 mm MaPMTs with 64 channels. 

Reviewer: ```suggestion
The outer area of RICH 2 is covered with 50.8 x 50.8 mm MaPMTs with 64 channels.
```

## MiniDAQ {#MiniDAQ}

> A readout system development stage server that hosts «timing and fast control system:TFC» and «data acquisition:DAQ» systems, see [LHCb MiniDAQ control system](https://cds.cern.ch/record/2702137/files/10.1051_epjconf_201921401005.pdf).

Reviewer: ```suggestion
A readout system development stage server that hosts «timing and fast control system:TFC» and «data acquisition:DAQ» systems, see the [MiniDAQ control system](https://cds.cern.ch/record/2702137/files/10.1051_epjconf_201921401005.pdf) document.
```
